README.md

```diff
@@ -1,11 +1,270 @@
-# Agora DRep Effect integration
+# Agora DRep Governance Effects
 
-Agora DRep Effect is an extension of the Agora project which allows a DAO to delegate their
-treasuries to a dRep and cast a vote for certain Cardano Governance Actions.
-This repository contains the Plutarch scripts for the Agora DRep effect extension.
-Find the references to this project below:
+## Introduction
 
-- [Specification](./specification)
-- [Haddock documentation](https://mlabs-haskell.github.io/agora-drep/haddock/)
-- [Agora repository](https://github.com/Liqwid-Labs/agora)
-- [Agora specification](https://liqwid.notion.site/Agora-Specs-Overview-fd7df78313cf4dc0b1522cb9260b77d1)
+This repository contains the source code for a new set of Plutus scripts that
+allow the [the Agora governance system](https://github.com/Liqwid-Labs/agora)
+to participate in Cardano governance proposals (see
+[Cardano Governance](https://cardano.org/governance/)).
+
+The general use case these scripts enable is the creation and handling of fully
+automated, on-chain DReps that can vote on _pre-existing_ Cardano Governance
+proposals (the creation of proposals is not supported).
+
+The policy that decides when or how the DRep votes is not given, which allows
+an Agora-based system to implement any suitable policy they desire. There already is
+an implementation of an automatic DRep for Agora DAOs created with
+[Clarity](https://www.clarity.vote/). See the [references](#references) at the end
+of the page for more information on this.
+
+In this document, an overview of the different scripts is given as well as instructions
+on how to deploy them. At the end, two use cases are explained and references are provided
+for more documents and websites of interest.
+
+## Keywords
+
+The following terms are used in the document:
+
+* _Agora system_: any governance system implemented using the set of governance
+  modules provided by [Agora](https://github.com/Liqwid-Labs/agora/tree/staging).
+
+* _Agora DAO_: A specific Agora system implemented by [Clarity](www.clarity.vote).
```

Comment:
[Clarity](www.clarity.vote) is acting like a relative link (broken). Add the `https://` protocol to fix it.